lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -781,7 +782,17 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
                 "passphrase": #{fixturePassphrase}
             }|]
                         )
-            sourceBalance <- eventually "Source wallet balance is correct." $ do
+
+            let transactions :: IO String
+                transactions = do
+                    (_ , txs) <- unsafeRequest @[ApiTransaction n] ctx
+                        (Link.listTransactions @'Shelley sourceWallet) Empty
+                    pure $
+                        "Source wallet balance is not correct. "
+                        <> "Transactions in the wallet are: "
+                            <> show (length txs)
```

Comment:
In the event of a failure, we probably want more information than `length txs` — could you `show txs` (or equivalent) here?

Reply:
I thought about that, but it's 200 txs. Wasn't the point just to know if they are not 200?